pullupclub/views.py

```diff
@@ -1,3 +1,6 @@
 from django.shortcuts import render
+from django.http import HttpResponse
 
```

Comment:
You got ahead of me here which is cool. Instead of doing this create `index.html` which extends base and serve that up from here.

Reply:
Done!